web/src/user_group_popover.ts

```diff
@@ -129,8 +130,15 @@ export function toggle_user_group_info_popover(
                     displayed_members,
                 };
                 instance.setContent(ui_util.parse_html(render_user_group_info_popover(args)));
+                $("body").on("click", ".user-group-info-popover .go-to-group-settings", () => {
+                    const group_visibility = group.deactivated
+                        ? FILTERS.DEACTIVATED_GROUPS
+                        : FILTERS.ACTIVE_GROUPS;
+                    update_displayed_groups(group_visibility);
+                });
```

Comment:
I think the code to set appropriate filter should be in `change_state` like how we set "Your groups" or "All groups" tab.